### PR TITLE
Adding emits options on the way to Vue 3

### PR DESCRIPTION
**Pull request by ktuite**
Cherry-pick of this commit from the massive Vue3 PR https://github.com/getodk/central-frontend/pull/526/commits/330c708a277498f501782eedf2842813988fe7e8

### CODE REVIEW

File: src/components/project/form-access.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@ except according to the terms contained in the LICENSE file.
     <template v-if="dataExists">
       <project-form-access-table :changes-by-form="changesByForm"
         @update:state="updateState"
-        @update:field-key-access="updateFieldKeyAccess"
+        @update:fieldKeyAccess="updateFieldKeyAccess"
```
Comment (matthew-white) [inline src/components/project/form-access.vue:37]: Actually, would you mind renaming this back to `@update:field-key-access` here? Just here in the template and in the template in `ProjectFormAccessTable`, but not anywhere else in the JS (the change works well in the JS). I think making this camel-case here was a mistake on my part: I ended up reverting it after the Vue 3 linter yelled at me.

File: src/components/audit/filters/action.vue [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default {
       required: true
     }
   },
+  emits: ['input'],
```
Comment (matthew-white) [inline src/components/audit/filters/action.vue:37]: Could you change all instances of `'input'` in the `emits` option to the (somewhat verbose&hellip;) `'update:modelValue'` (which `v-model` uses in Vue 3)?

Comment (ktuite) [reply src/components/audit/filters/action.vue]: done!

File: src/components/project/form-access.vue (hunk 1 above)
Comment (ktuite) [reply src/components/project/form-access.vue]: done!

**Comment by matthew-white** [comment]
The link in the commit message doesn't seem to work anymore, but here's the latest link to the relevant section of the migration guide: https://v3-migration.vuejs.org/breaking-changes/emits-option.html

**Review by matthew-white — APPROVED** [review]

**Pull request merged**